src/msg/async/dpdk/dpdk_rte.cc

```diff
@@ -151,4 +154,14 @@ namespace dpdk {
     return memsize;
   }
 
+  void eal::exit()
+  {
+    if (initialized == false || exit_thread)
```

Comment:
i'd say it's a bug to call this method if `! initialized` or `exit_thread`. so can we instead
```c++
assert(initialized);
assert(!stopped);
```
?

Reply:
ok